tardis/tests/tests_slow/test_w7.py

```diff
@@ -178,34 +164,21 @@ def test_montecarlo_properties(self):
                 self.result.montecarlo_nu)
 
     def test_shell_temperature(self, plot_object):
-        try:
-            assert_quantity_allclose(
-                self.reference['t_rads'],
-                self.result.t_rads)
-        except Exception as e:
-            plot = self.plot_t_rads(has_passed=False)
-            raise e
-        else:
-            plot = self.plot_t_rads(has_passed=True)
-
-        plot_object.add(plot, "t_rads")
-
-    def plot_t_rads(self, has_passed):
+        assert_quantity_allclose(
+            self.reference['t_rads'],
+            self.result.t_rads)
+
+        plot_object.add(self.plot_t_rads(), "t_rads")
```

Comment:
same here:
plot first then assert
otherwise we won't get a plot if there is an assert.